esp/src/eclwatch/Utility.js

```diff
@@ -21,9 +21,10 @@
     }
 
     function parseXML (val) {
+        var xmlDoc;
```

Comment:
Can you please give an explanation as to why this `var` has to be moved to a different scope? So we can accommodate future files

Reply:
Thanks.